__tests__/lint/assertions-custom-function-options/snapshot.js

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E lint assertions-custom-function-options 1`] = `
+
+validating /openapi.yaml...
+[1] openapi.yaml:36:16 at #/paths/~1pet~1findByStatus/post/summary
+
+Operation summary should start with an active verb
+
+34 | post:
+35 |   operationId: example-post
+36 |   summary: summary looooong
+   |            ^^^^^^^^^^^^^^^^
+37 |   tags:
+38 |     - foo
+
+Error was generated by the operation-summary-length assertion rule.
+
+
+[2] openapi.yaml:36:16 at #/paths/~1pet~1findByStatus/post/summary
+
+Operation summary should start with an active verb
+
+34 | post:
+35 |   operationId: example-post
+36 |   summary: summary looooong
+   |            ^^^^^^^^^^^^^^^^
+37 |   tags:
+38 |     - foo
+
+Error was generated by the operation-summary-length assertion rule.
+
```

Comment:
I think we would want these somehow combined into one problem message?

Reply:
Yes. I'm working currently on it